libs/api/database/src/lib/content/schemas/poetry-content.schema.ts

```diff
@@ -30,7 +31,7 @@ export class PoetryContentDocument extends Document implements PoetryContent {
         category: { type: String, enum: Object.keys(WorkKind), required: true },
         form: { type: String, enum: Object.keys(PoetryForm), required: true },
         collection: { type: Boolean, default: false },
-        fandoms: { type: [String], default: null },
+        fandoms: { type: [MongooseSchema.Types.ObjectId], default: null, ref: 'TagsDocument' },
```

Comment:
Let's actually have a bit of a discussion about this: as implemented, Tags can work for Fandoms, User Tags, or Warning tags. Do we want a separate field on each bit of content that's just a subset of the actual Tags field? I'm not really sure. Would love input.

Reply:
Sorry, I'm having trouble parsing your question. Can you give an example?

Reply:
the problem with doing that is how would we separate out which tags are which? we could use the tags' `kind`, but that creates additional overhead on the client side to parse out and arrange the tags in our desired structure, e.g. fandom tags in the top tag row, content warning tags bottom left, etc

Reply:
That feels like a small price to pay in exchange for a much simpler data model in the database and on the backend. Maybe we expose them to the client as three different types (i.e. actually separate interfaces), but I think it makes sense to treat them all as the same thing.

Worst-case scenario, a handful of `switch` statements in the client doesn't feel like a _terrible_ imposition?